src/styles/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 0.3s;
```

Comment:
you have to set a correct property in transition to specify on which property you want to add transition